src/librustc_typeck/check/regionck.rs

```diff
@@ -544,7 +544,7 @@ fn visit_expr(rcx: &mut Rcx, expr: &ast::Expr) {
                 if tcx.sess.has_errors() {
                     // cannot run dropck; okay b/c in error state anyway.
                 } else {
-                    tcx.sess.span_bug(expr.span, "cat_expr_unadjusted Errd");
+                    tcx.sess.delay_span_bug(expr.span, "cat_expr Errd");
```

Comment:
don't change the message here; it is still a call to `cat_expr_unadjusted` up above, which is info I wanted to capture in the ICE message.